src/main/java/com/example/nlern/models/Question.java

```diff
@@ -1,86 +1,41 @@
 package com.example.nlern.models;
-import jakarta.persistence.*;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.ManyToOne;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
 
 import java.util.List;
 
 @Entity
 @Table(name = "questions")
+@Getter @Setter
 public class Question {
-
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private int id;
 
-
     private String question;
-
     private boolean isFlags;
```

Comment:
What doest isFlags mean?

Reply:
determines whether the question will have multiple correct answers.

Reply:
determines whether the question will have multiple answers